tests/test_rm/integration/test_rm_int.py

```diff
@@ -0,0 +1,178 @@
+import os
+import glob
+import shutil
+import radical.pilot as rp
+from radical.pilot.agent import rm as rpa_rm
+from radical.pilot.agent.rm.pbspro import PBSPro
+
+try:
+    import mock
+except ImportError:
+    from unittest import mock
+
+# ------------------------------------------------------------------------------
+#
+def setUp(resource):
+
+    session      = rp.Session()
+    cfg          = session.get_resource_config(resource=resource)
```

Comment:
Can I ask that the sessions are mocked? I believe that reading just the configuration should be enough. @andre-merzky can we read a config file somehow by providing the resource label without having to create a session?

Reply:
Well, one can read the resource config via `ru.read_json(filename)`.  The filename is easy to derive from the resource label, and so is the config section to be used.  But we don't have a separate method.

Reply:
Well, the session has a `_load_resource_configs(self)` method - but that expects an initialized session to exist...